.NET Interactive: Add_try_dotnet_batch_31a

v-mepa opened the pull request:
fyi @mairaw, I am going back to creating multiple PR's for each batch. You can review them as per your convenience. 


## Add_try_dotnet_batch_31a


S No | URL | PageViews | BounceRate | Try .NET Status | Comments
-- | -- | -- | -- | -- | --
1 | https://docs.microsoft.com/en-us/dotnet/api/system.convert.todecimal | 8945 | 14 | Batch 31 | Except 3, 7, 8, 12, 15, 17,   18. Not runnable or depend on each other
2 | https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.numberformat | 707 | 9 | Batch 31 |  
3 | https://docs.microsoft.com/en-us/dotnet/api/system.random.sample | 697 | 9 | Batch 31 |  
4 | https://docs.microsoft.com/en-us/dotnet/api/system.type.isclass | 676 | 14 | Batch 31 |  
5 | https://docs.microsoft.com/en-us/dotnet/api/system.bitconverter.toint16 | 667 | 13 | Batch 31 |  

opbld33 commented:
Docs Build status updates of commit _[2dd447f](https://github.com/v-mepa/dotnet-api-docs/commits/2dd447fd536787c8a3228c06e9da3d36961597e3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Globalization/CultureInfo.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System.Globalization/CultureInfo.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/BitConverter.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/BitConverter.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.bitconverter?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Convert.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Convert.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.convert?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Random.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Random.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.random?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Type.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Type.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.type?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C16%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202103152336051934-5438%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KY3iMDMp2a10lHB9V7RjWlPGJTDT6W%2F5q9vof0kigpc%3D&st=2021-03-15T23%3A56%3A38Z&se=2021-04-16T00%3A01%3A38Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6726,9 +6726,9 @@
- [!code-csharp[System.Convert Snippets#15](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs#15)]
- [!code-vb[System.Convert Snippets#15](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Convert Snippets/VB/system.convert snippet.vb#15)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Convert Snippets/CPP/system.convert snippet.cpp" id="Snippet15":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" interactive="try-dotnet-method" id="Snippet15":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" id="Snippet15":::
```

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6986,13 +6986,13 @@
 
- [!code-csharp[System.Convert.ToDecimal2#10](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs#10)]
- [!code-vb[System.Convert.ToDecimal2#10](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.convert.todecimal2/vb/todecimal2.vb#10)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs" interactive="try-dotnet" id="Snippet10":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs" id="Snippet10":::
```

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6986,13 +6986,13 @@
 
- [!code-csharp[System.Convert.ToDecimal2#11](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs#11)]
- [!code-vb[System.Convert.ToDecimal2#11](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.convert.todecimal2/vb/todecimal2.vb#11)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs" interactive="try-dotnet-method" id="Snippet11":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.convert.todecimal2/cs/todecimal2.cs" id="Snippet11":::
```

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6457,9 +6457,9 @@
- [!code-csharp[System.Convert Snippets#10](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs#10)]
- [!code-vb[System.Convert Snippets#10](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Convert Snippets/VB/system.convert snippet.vb#10)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Convert Snippets/CPP/system.convert snippet.cpp" id="Snippet10":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" interactive="try-dotnet-method" id="Snippet10":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" id="Snippet10":::
```

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6274,17 +6274,17 @@
- [!code-csharp[System.Convert Snippets#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs#5)]
- [!code-vb[System.Convert Snippets#5](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Convert Snippets/VB/system.convert snippet.vb#5)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Convert Snippets/CPP/system.convert snippet.cpp" id="Snippet5":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" interactive="try-dotnet-method" id="Snippet5":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" id="Snippet5":::
```

mairaw commented on xml/System/Convert.xml:
```diff
@@ -6124,9 +6124,9 @@
- [!code-csharp[System.Convert Snippets#17](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs#17)]
- [!code-vb[System.Convert Snippets#17](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Convert Snippets/VB/system.convert snippet.vb#17)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Convert Snippets/CPP/system.convert snippet.cpp" id="Snippet17":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" interactive="try-dotnet-method" id="Snippet17":::
```
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Convert Snippets/CS/system.convert snippet.cs" id="Snippet17":::
```

mairaw reviewed (changes requested):
Thanks @v-mepa. Check https://review.docs.microsoft.com/en-us/dotnet/api/system.convert.todecimal?branch=pr-en-us-5438&view=net-5.0#code-try-32 to see which ones were missed. I commented on the ones that shouldn't have .NET Interactive but there a couple of ones that should be that aren't (review the examples number and let me know if I got the numbers wrong).

v-mepa commented:
There are 18 examples and I counted the examples not the samples, my mistake. I have updated the missing samples and removed interactive from the ones mentioned in the comment.

opbld32 commented:
Docs Build status updates of commit _[870caa4](https://github.com/v-mepa/dotnet-api-docs/commits/870caa42fdf3562591758dddf29253ef06eb9250)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Globalization/CultureInfo.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System.Globalization/CultureInfo.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/BitConverter.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/BitConverter.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.bitconverter?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Convert.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Convert.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.convert?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Random.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Random.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.random?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |
[xml/System/Type.xml](https://github.com/v-mepa/dotnet-api-docs/blob/add_try_dotnet_to_batch31/xml/System/Type.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.type?view=dotnet-uwp-10.0&branch=pr-en-us-5438) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C17%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202103171838082202-5438%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IYzQtC1IRiHZS25sIZMXHQExKhhsxnMNoH%2F31I5R7sk%3D&st=2021-03-17T19%3A00%3A00Z&se=2021-04-17T19%3A05%3A00Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
